commercetools-models/src/main/java/io/sphere/sdk/products/ProductDraftBuilder.java

```diff
@@ -16,7 +16,7 @@
 public final class ProductDraftBuilder extends ProductDraftBuilderBase<ProductDraftBuilder> {
 
 
-    ProductDraftBuilder(Set<ResourceIdentifier<Category>> categories, @Nullable CategoryOrderHints categoryOrderHints, @Nullable LocalizedString description, @Nullable String key, ProductVariantDraft masterVariant, @Nullable LocalizedString metaDescription, @Nullable LocalizedString metaKeywords, @Nullable LocalizedString metaTitle, LocalizedString name, ResourceIdentifier<ProductType> productType, @Nullable Boolean publish, SearchKeywords searchKeywords, LocalizedString slug, @Nullable Reference<State> state, @Nullable Reference<TaxCategory> taxCategory, List<ProductVariantDraft> variants) {
+    ProductDraftBuilder(Set<ResourceIdentifier<Category>> categories, @Nullable CategoryOrderHints categoryOrderHints, @Nullable LocalizedString description, @Nullable String key, @Nullable ProductVariantDraft masterVariant, @Nullable LocalizedString metaDescription, @Nullable LocalizedString metaKeywords, @Nullable LocalizedString metaTitle, LocalizedString name, ResourceIdentifier<ProductType> productType, @Nullable Boolean publish, SearchKeywords searchKeywords, LocalizedString slug, @Nullable Reference<State> state, @Nullable Reference<TaxCategory> taxCategory, List<ProductVariantDraft> variants) {
```

Comment:
We try to follow the "Boy Scout Rule": we should leave the code cleaner than we found it.  So can you please add the missing `final` modifiers to the parameters? 😄

Reply:
@katmatt you mean all the constructor arguments?
Also, i'd like to format/indent the arguments list as multiline, but usually it requires separate PR. Since i'll modify all the arguments, could i also format/indent them to the page margin (about 120 columns)?

Reply:
with all `final` now the constructor arguments list is 700 columns.

Reply:
@andrii-kovalenko-ct Yes, please format/indent them as part of this PR.

Reply:
c738e9aa60